engine/index.js

```diff
@@ -298,6 +299,49 @@ function populateCanIUsePercent(canIUseData, features) {
   });
 }
 
+const storedFields = ['firefox_status', 'spec_status', 'opera_status',
+                      'webkit_status', 'ie_status'];
+function checkForNewData(features) {
+  const client = redis.createClient(process.env.REDISCLOUD_URL, { no_ready_check: true });
```

Comment:
Wouldn't it be easier to use a single client instance to avoid opening/closing connections?